apps/site/vercel-lib.sh

```diff
@@ -13,6 +13,9 @@ lib_files=(
    libxmlsec1.so.1.2.20
    libxslt.so.1
    libxslt.so.1.1.28
+   libexslt.so.0
+   libgcrypt.so.11
+   libgpg-error.so.0
```

Comment:
Thought: I'm confused how this resolves the issue (or why we're even encountering the issue) when each of these are symlinks to the actual library file with the full version number (e.g. `libexslt.so.0 -> libexslt.so.0.8.17`), but if it works, so be it 🤷.

Reply:
On the surface, it appears the libraries aren't needed, but let's include the actual library files anyway to avoid potential issues down the line. See the lines above for how we specify the symlink and the actual library file.